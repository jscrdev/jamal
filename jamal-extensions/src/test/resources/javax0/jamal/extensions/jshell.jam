{@sep (( )) }((@JShell

void hello(){
    System.out.println("hello " + b);
}

void hallo(){
    System.out.println("hallo " + a);
}
))\
((@script hello/JShell(a,b)=hallo();
hello() ))\
((@script helloBlock/JShell(a,b)=hallo();
hello() ))\
((hello/Misi/Matyi))
((hello/"Misi
Matyi
Nagyi
Anya \ Apa"/Eszti))

((helloBlock/Misi/Matyi))