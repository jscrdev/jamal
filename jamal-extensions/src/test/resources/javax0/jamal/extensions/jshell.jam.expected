hallo Misi
hello Matyi

hallo "Misi
Matyi
Nagyi
Anya \ Apa"
hello Eszti


hallo Misi
hello Matyi
